app/api/csv_columns.py

```diff
@@ -141,7 +141,6 @@ def __init__(self, name):
     CoreData.totalTestsAntigen,
     CoreData.positiveTestsAntigen,
     CoreData.negativeTestsAntigen,
-    literal_column("''").label('_posNeg'),  # Fake Column
```

Comment:
on board with this, but let's hold off on any code changes until tomorrow after we've launched? We'll be saner with a code freeze